package-lock.json

```diff
@@ -0,0 +1,11 @@
+{
+  "requires": true,
+  "lockfileVersion": 1,
+  "dependencies": {
+    "esm": {
```

Comment:
if `esm` is needed, please add to `devDependencies` and regenerate `package-lock.json`, and remember to lock version